Extension/Xamarin.Forms.HotReload.Extension/ExtensionCore.cs

```diff
@@ -128,7 +128,7 @@ private void OnInfoBarResult(Task<InfoBarActionType> resultTask)
 
         private void OnEnviromentSolutionClosed(object sender, EventArgs e)
         {
-            _environmentService.DocumentSaved -= OnEnviromentDocumentSaved;
+            DisableExtension();
```

Comment:
do you think it's a proper solution indeed?

seems it would be better to add _isRunning filed and prevent running more than one simultaneously 

https://github.com/AndreiMisiukevich/HotReload/blob/9ac23ec61d4079b300aaace1c91d8b0b04701ea2/Extension/Xamarin.Forms.HotReload.Extension/Helpers/HotReloadClientsHolder.cs

Reply:
I fixed the issue that the reloader is running after solution close like I reported. You can for sure check if it is running and skip the button state toggling to leave the listener running after closing a solution. But I don't want to introduce design changes. This must be done by @iBavtovich .